### ISSUE TITLE
I can not find smux-anki-convert-1.0.jar

### ISSUE THREAD

**Issue by szparag3** [ggodlewski/smux-anki-converter#4]
HI grzegorz 

Could you explain me how to convert kein problem course if without jar file.

Regards

Peter

**Comment by albertly** [comment]
Yes, this is a problem. However, once the author has published a link of version 1.0 and 1.1 . I've found this
https://github.com/downloads/ggodlewski/smux-anki-converter/smux-anki-convert-1.4.jar

**Comment by jigitjigit1** [comment]
Can anyone, please, compile version 1.7 or explain how to use it. It seems to be the newest release https://github.com/ggodlewski/smux-anki-converter/releases
